examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<blockquote>BATCH_SIZE = 64</blockquote><blockquote>dataset = MNIST()</blockquote><blockquote>config = dict(model=EncoderDecoder)</blockquote><p>Use same way to initilize them in all notebooks, usually these three lines were in different cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>